examples/convert_mri_and_trans.py

```diff
@@ -0,0 +1,162 @@
+"""
+==========================================================================
+Save and load T1-weighted MRI scan along with anatomical landmarks in BIDS
+==========================================================================
+
+When working with MEEG data in the domain of source localization, we usually
+have to deal with aligning several coordinate systems, such as the coordinate
+systems of ...
+
+- the head of a study participant
+- the recording device (in the case of MEG)
+- the anatomical MRI scan of a study participant
+
+The process of aligning these frames is also called coregistration, and is
+performed with the help of a transformation matrix, called ``trans`` in MNE.
+
+In this tutorial, we show how ``MNE-BIDS`` can be used to save a T1 weighted
+MRI scan in BIDS format, and to encode all information of the ``trans`` object
+in a BIDS compatible way.
+
+Finally, we will automatically reproduce our ``trans`` object from a BIDS
+directory.
+
+See the documentation pages in the MNE docs for more information on
+`source alignment and coordinate frames <mne_source_coords_>`_
+
+.. note:: For this example you will need to install ``matplotlib`` and
+          ``nilearn`` on top of your usual ``mne-bids`` installation.
+
+"""
+# Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+# License: BSD (3-clause)
+
+###############################################################################
+# We are importing everything we need for this example:
+
+import os.path as op
+import shutil as sh
+
+import numpy as np
+import matplotlib.pyplot as plt
+from nilearn.plotting import plot_anat
+import mne
+from mne.datasets import sample
+from mne.source_space import head_to_mri
+
+from mne_bids import (write_raw_bids, make_bids_basename, write_anat,
+                      get_head_mri_trans)
+from mne_bids.utils import print_dir_tree
+
+###############################################################################
+# We will be using the `MNE sample data <mne_sample_data_>`_ and write a basic
+# BIDS dataset. For more information, you can checkout the respective
+# :ref:`example <ex-convert-mne-sample>`.
+
+data_path = sample.data_path()
+event_id = {'Auditory/Left': 1, 'Auditory/Right': 2, 'Visual/Left': 3,
+            'Visual/Right': 4, 'Smiley': 5, 'Button': 32}
+raw_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')
+events_data = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw-eve.fif')
+output_path = op.abspath(op.join(data_path, '..', 'MNE-sample-data-bids'))
+if op.exists(output_path):
+    sh.rmtree(output_path)
+raw = mne.io.read_raw_fif(raw_fname)
+sub = '01'
+ses = '01'
+task = 'audiovisual'
+run = '01'
+bids_basename = make_bids_basename(subject=sub, session=ses, task=task,
+                                   run=run)
+write_raw_bids(raw, bids_basename, output_path, events_data=events_data,
```

Comment:
It's a bit unfortunate that we call it `output_path` in `write_raw_bids` and `bids_root` in `read_raw_bids`

Reply:
I agree, perhaps we should make an issue an homogenize it in the future

Reply:
see: https://github.com/mne-tools/mne-bids/issues/213